compiler/job.go

```diff
@@ -199,3 +201,18 @@ func VectorFilterCompile(rctx *runtime.Context, query string, src *data.Source,
 	}
 	return kernel.NewBuilder(rctx, src).BuildVamToSeqFilter(f.Expr, poolID, commitID)
 }
+
+func Describe(ctx context.Context, query string, includes []string, src *data.Source, head *lakeparse.Commitish) (*describe.Info, error) {
+	seq, sset, err := parser.ParseZed(includes, query)
+	if err != nil {
+		return nil, err
+	}
+	entry, err := semantic.Analyze(ctx, seq, src, head)
+	if err != nil {
+		if list, ok := err.(parser.ErrorList); ok {
+			list.SetSourceSet(sset)
+		}
+		return nil, err
+	}
+	return describe.Analyze(ctx, entry, src, head)
+}
```

Comment:
I don't see any benefit in putting this here so how about keeping it as compiler/describe.Analyze and adding compiler/describe.AnalyzeDAG?

Reply:
sure